cli/src/cmd/forge/script/executor.rs

```diff
@@ -201,14 +272,19 @@ impl ScriptArgs {
     ) -> ScriptRunner {
         trace!("preparing script runner");
         let env = script_config.evm_opts.evm_env().await;
+        let url = script_config.evm_opts.fork_url.clone().unwrap_or_default();
```

Comment:
this could be a pitfall if `fork_url` ends up being None

should we use an error instead?